internal/resources/data_source_loadbalancer.go

```diff
@@ -1,88 +0,0 @@
-// (C) Copyright 2022 Hewlett Packard Enterprise Development LP
-
-package resources
-
-import (
-	"context"
-
```

Comment:
I'm assuming you will add DS for LB in another review

Reply:
yes